plugins/micromega/g_micromega.mlg

```diff
@@ -10,9 +10,9 @@
 (*                                                                      *)
 (* Micromega: A reflexive tactic using the Positivstellensatz           *)
 (*                                                                      *)
-(* * Mappings from Coq tactics to Caml function calls                   *)
+(* * Mappings from Rocq tactics to Caml function calls                  *)
 (*                                                                      *)
-(*  Frédéric Besson (Irisa/Inria) 2006-2008			        *)
+(*  Frédéric Besson (Irisa/Inria) 2006-2008			                    *)
```

Comment:
How about avoiding tabs here altogether?

Reply:
Yes, I tried to do that but forgot to commit and I'm too lazy to open another PR just for that.